plugins/wallet_plugin/wallet_manager.cpp

```diff
@@ -275,6 +276,22 @@ void wallet_manager::own_and_use_wallet(const string& name, std::unique_ptr<wall
    wallets.emplace(name, std::move(wallet));
 }
 
+void wallet_manager::start_lock_watch(std::shared_ptr<boost::asio::deadline_timer> t)
+{
+   t->async_wait([t, this](const boost::system::error_code& /*ec*/)
+   {
+      struct stat statbuf;
+      int rc = stat(lock_path.string().c_str(), &statbuf);
```

Comment:
Can this be made in to a boost filesystem call instead?

Reply:
Converted to boost::filesystem.